packages/terra-site/src/examples/table/examples/MultiRowSelectableTable.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
It doesn't look like this file is being used anywhere...

Reply:
You're right. [Deleted this file](https://github.com/cerner/terra-core/pull/970/files/84e8501009018d54655e3b677f2e26a1465beab9..b75a40549d698748c69fd3e2638f5aac789464be#diff-acdeacd3cc8c3c673be8201657fcf4fdL1)